src/emscripten-optimizer/simple_ast.h

```diff
@@ -84,6 +84,7 @@ struct Arena {
   int arr_index;
 
   Arena() : index(0), arr_index(0) {}
+  ~Arena();
```

Comment:
Sorry, should have elaborated: would have been nicer to have a container instead, so you can use the default dtor.